runtime.go

```diff
@@ -515,6 +515,9 @@ func (r *Runtime) MemUsage(ctx *MemUsageContext) (memUsage uint64, newMemUsage u
 			if err != nil {
 				return memUsage, newMemUsage, err
 			}
+			if exceeded := ctx.MemUsageLimitExceeded(memUsage); exceeded {
```

Comment:
[q] Do we need to add the same check after Line 503 for `r.globalObject`?

Reply:
I definitely can, it might not be as effective but it doesn't hurt, let me make the change!